src/stratis_cli/_actions/_data.py

```diff
@@ -29,6 +29,9 @@
 
 from ._constants import DBUS_TIMEOUT_SECONDS
 
+MAXIMUM_STRATISD_VERSION = (1, 0, 1)
+REQUIRED_STRATISD_VERSION = (1, 0, 1)
```

Comment:
suggest `MINIMUM_STRATISD_VERSION`? And, why should this not be 1.0.0?

Reply:
> suggest `MINIMUM_STRATISD_VERSION`? And, why should this not be 1.0.0?

Yes. ```MINIMUM``` is better. About the second question, I was just guessing wildly. If you think it's 1.0.0, that's what you can have. Eventually this will be the biggest burden brought on by the code in this PR, keeping track of the minimum and maximum version. Are we all up for it?

Reply:
If we adhere to semantic versioning we should only need to worry about a minimum version

Reply:
> If we adhere to semantic versioning we should only need to worry about a minimum version

We have explicitly not stabilized our D-Bus API yet, so I think maximum is still worth keeping track of.